### PR TITLE
Make rad project work with existing git repos.

### PR DESCRIPTION
**Pull request by jameshaydon**

### CODE REVIEW

**Comment by clevinson** [comment]
glad to see this getting worked on!

**Comment by clevinson** [comment]
Functionality wise, it works for me (just ran from this branch on a test project with an existing remote). Though still has all the same prompts as before (you probably already know this...)

**Comment by jameshaydon** [comment]
@clevinson I have added a new option, let me know if you think the prompts make sense now.

File: bin/rad-project [hunk 1]
```diff
@@ -82,20 +82,30 @@
...
+    (def current-remote (get-git-config! "remote.origin.url"))
     (def question
       (unlines
-       ["? What kind of repository would you like to use?"
+       (<>
+        ["? What kind of repository would you like to use?"
         "1. New peer-to-peer repository (git-ipfs **EXPERIMENTAL**)"
         "2. Add your own remote (e.g.: github / gitlab / ...)"
-        "    (see radicle.xyz/docs/storage for more info)\n"]))
+         "    (see radicle.xyz/docs/storage for more info)\n"]
```
Comment (jkarni) [inline bin/rad-project]: Shouldn't this line be after 3?

**Pull request merged**